src/main/kotlin/com/hibob/academy/employeeFeedback/dao/FeedbackDao.kt

```diff
@@ -72,10 +75,41 @@ class FeedbackDao @Inject constructor(
             .execute() > 0
     }
 
-    fun getFeedbacks(companyId: Long) : List<Feedback> {
-        return sql.select()
+    fun getFeedbacksByFilters(companyId: Long, filters: FilterFeedbackRequest): List<Feedback> {
```

Comment:
We want to avoid using mutable collections. Also I expect a generic solution. If later on we would like to have more filters, I expect to deal with it in 1 place